knowledge-base/axel-op/googlejavaformat-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: "Google Java Format"
+github-token:
+  action-input:
+    input: githubToken
+  permissions:
+    contents: read
+    contents-reason: to get releases #Checkout: https://github.com/axel-op/googlejavaformat-action#githubtoken
```

Comment:
is this also for rate limit?

Reply:
@varunsh-coder yes, it is being use for overcoming rate limiting